manageiq-api-common.gemspec

```diff
@@ -29,9 +29,10 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "more_core_extensions"
 
   # For ManageIQ::API::Common::GraphQL
-  spec.add_runtime_dependency "graphql",         "~> 1.7"
-  spec.add_runtime_dependency "graphql-batch",   "~> 0.3.8"
-  spec.add_runtime_dependency "graphql-preload", "~> 1.0"
+  spec.add_runtime_dependency "graphql",         "~> 1.9"
+  spec.add_runtime_dependency "graphql-batch",   "~> 0.4"
+  spec.add_runtime_dependency "graphql-preload", "~> 2.0.1"
```

Comment:
Do they follow semver?  If so I would prefer to use `~> 2.0` with an additional `>= 2.0.1` if that's important.